### DISCUSSION TITLE
Doubt in the `INVARIANT TESTING` in `Section 3: Develop a DeFi Protocol , 21. Create the mint handler 4:41`

### DISCUSSION THREAD

**Discussion by Basha-dude** [Cyfrin/foundry-full-course-cu#2171] [General]
why used this 
```solidity
  amount = bound(amount,0,maxDscToMint);
   if(amount ==0) {
           return;
        }
```
in the `handler.t.sol::mintDsc`,why not this 

```solidity
  amount = bound(amount,1,maxDscToMint);
```

**Comment by EngrPips** [comment]
Hello @Basha-dude, I think you can use either of the two as one will return when the chosen bound is `0` and one will never choose `0` and the process will always proceed either both methods prevent reverts I believe.
